### PR TITLE
Only return one IP address from all three returns

### PR DESCRIPTION
**Pull request by mbandizzle**
This handling mirrors the setup in the Sentry module: https://github.com/coldbox-modules/sentry/blob/385e1abf5c38b86fb33b86a277cc0e83139230ad/models/SentryService.cfc#L752-L768

### CODE REVIEW

File: interceptors/Security.cfc [hunk 1]
```diff
@@ -729,15 +729,16 @@ component accessors="true" extends="coldbox.system.Interceptor" {
...
-			return headers[ "x-cluster-client-ip" ];
+		// When going through a proxy, the IP can be a delimtied list, thus we take the last one in the list
+		
+		if ( structKeyExists( headers, "x-cluster-client-ip" ) ){
+			return trim( listLast( headers[ "x-cluster-client-ip" ] ) );
 		}
-		if ( structKeyExists( headers, "X-Forwarded-For" ) ) {
-			return headers[ "X-Forwarded-For" ];
+		if ( structKeyExists( headers, "X-Forwarded-For" ) ){
+			return trim( listFirst( headers[ "X-Forwarded-For" ] ) );
```
Comment (lmajano) [inline interceptors/Security.cfc:738]: Why take the first one here, and not the last one?

File: interceptors/Security.cfc [hunk 2]
```diff
@@ -729,15 +729,16 @@ component accessors="true" extends="coldbox.system.Interceptor" {
...
 		}
 
-		return len( CGI.REMOTE_ADDR ) ? trim( listFirst( CGI.REMOTE_ADDR ) ) : "127.0.0.1";
+		return len( cgi.remote_addr ) ? trim( listFirst( cgi.remote_addr ) ) : "127.0.0.1";
```
Comment (lmajano) [inline interceptors/Security.cfc:741]: Same question, why first and not last?

File: interceptors/Security.cfc (hunk 1 above)
Comment (mbandizzle) [reply interceptors/Security.cfc:738]: https://support.cloudflare.com/hc/en-us/articles/200170986-How-does-Cloudflare-handle-HTTP-Request-headers-

X-Forwarded-For

Maintains proxy server and original visitor IP addresses. If there was no existing X-Forwarded-For header in the request sent to Cloudflare, X-Forwarded-For has an identical value to the CF-Connecting-IP header:

Example:

X-Forwarded-For: 203.0.113.1

If an X-Forwarded-For header was already present in the request to Cloudflare, Cloudflare appends the IP address of the HTTP proxy to the header:

Example:

X-Forwarded-For: 203.0.113.1,198.51.100.101,198.51.100.102

In the examples above, 203.0.113.1 is the original visitor IP address and 198.51.100.101 and 198.51.100.102 are proxy server IP addresses provided to Cloudflare via the X-Forwarded-For header.

File: interceptors/Security.cfc (hunk 2 above)
Comment (mbandizzle) [reply interceptors/Security.cfc:741]: Similar to the note above.  This one's more just based on experience that it seems to be the case.  First IP in a list is the original source address and then along the way any proxied address gets appended to the list.

**Pull request merged**